javascript-practice/src/javascripts/controllers/detail.js

```diff
@@ -0,0 +1,11 @@
+import HomeController from "./home";
+
+export default class DetailController extends HomeController {
+	constructor() {
+		super()
+
+		this.init();
+	}
+
+	init = () => { }
```

Comment:
What does this function do?

Reply:
I use it to connect views and models related to detail page